Swap first and last variable.py

```diff
@@ -0,0 +1,15 @@
+
+Data = input()
+a= list(Data)
+
+b = a[0]
+
+c= b[4]
+
+[b,c] = [c,b]
+
+
+
+print(Data)
```

Comment:
Incorrect login.
Data is declared and assigned to a, but swapping is not happening.